lambdas/src/services/session-service.ts

```diff
@@ -56,6 +61,14 @@ export class SessionService {
             throw new InvalidAccessTokenError();
         }
 
+        if (sessionItem.Items[0].authorizationCodeExpiryDate < new Date().getDate()) {
+            throw new AuthorizationCodeExpiredError();
+        }
+
+        if (sessionItem.Items[0].accessTokenExpiryDate < new Date().getDate()) {
```

Comment:
I think this is out of scope of this ticket, it is the Credential issue lambda that needs to check whether the accesstoken has expired which is further upstream (we haven't converted issuercredential lambda that why the method appeared to be missing, it is not i.e it is the java sessionService). Whereas the Accesstoken lambda needs to know if the authorizationCode created downstream i.e created when /Authorize endpoint was hit) is still valid

Reply:
OK - Removed it